lib/block-supports/layout.php

```diff
@@ -70,6 +70,14 @@ function gutenberg_get_layout_style( $selector, $layout ) {
 		$style .= 'gap: var( --wp--style--block-gap, 0.5em );';
 		$style .= 'flex-wrap: wrap;';
 		$style .= 'align-items: center;';
+		/**
+		 * Add this style only if is not empty for backwards compatibility,
+		 * since we intend to convert blocks that had flex layout implemented
+		 * by custom css.
+		 */
+		if ( ! empty( $layout['justifyContent'] ) ) {
+			$style .= "justify-content: {$layout['justifyContent']};";
+		}
```

Comment:
@youknowriad 's comment:
>I was expecting more changes here to retrieve the "default layout" from the block support config and apply it when the "layout" attribute is empty.

The passed `layout` has already checked the existence of default block layout.